src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -131,24 +107,28 @@ private void assertGaugeSnapshot(MetricSnapshot snapshot, double expectedValue,
         assertEquals(expectedLabels, datapoint.getLabels());
     }
 
-    private KafkaMetric buildMetric(String name, String group, double value) {
+    private MetricWrapper buildMetric(MetricName metricName, double value) {
```

Comment:
Should we call that method `newMetric()` so it matches `newNonNumericMetric()` and also the names used in `YammerMetricsCollectorTest`?

Reply:
Yes, changed to `newMetric()`